nephio-ansible-install/README.md

```diff
@@ -38,11 +38,13 @@ all:
     cloud_user: <username that is used to access the VM>
     github_username: <GitHub username>
     github_token: <GitHub personal access token>
-    github_organization: <optional, if a GitHub organization is used this should be filled out with the GitHub organization>
+    github_uri: github.com <replace this if you have a different github domain name in case of organization hosted github>
```

Comment:
This change is to allow other github domain as well, mainly in case of any organization's internal git hub.
We need to make similar change on the UI as well to allow different git hub domains.